ubo-common/src/main/resources/xsl/ubo-solr.xsl

```diff
@@ -50,7 +50,7 @@
     <xsl:apply-templates select="descendant::mods:originInfo" mode="solrField" />
     <xsl:apply-templates select="descendant::mods:relatedItem[@type='series']/mods:titleInfo" mode="solrField" />
     <xsl:apply-templates select="descendant::mods:name[@type='conference'][not(ancestor::mods:relatedItem[@type='references'])][1]" mode="solrField" />
-    <xsl:apply-templates select="descendant::mods:dateIssued[1][translate(text(),'1234567890','YYYYYYYYYY')='YYYY']" mode="solrField" />
+    <xsl:apply-templates select="mods:originInfo/mods:dateIssued[1][translate(text(),'1234567890','YYYYYYYYYY')='YYYY']" mode="solrField" />
     <xsl:apply-templates select="mods:relatedItem[@type='host']/mods:originInfo/mods:dateIssued[1][translate(text(),'1234567890','YYYYYYYYYY')='YYYY']" mode="solrField.host" />
     <xsl:apply-templates select="descendant::mods:identifier[@type]" mode="solrField" />
     <xsl:apply-templates select="descendant::mods:shelfLocator" mode="solrField" />
```

Comment:
That will only index the date if it is at the publicaton level.
There are also cases where the date is -only- in the relatedItem type host.
What we need is a priority:
- If there is a date at the publication level (mods:mods), use that.
- If not, use a date at the parent level (type host), or maybe even higher up.
I think we need a choose/when, that can not be done with a simple xpath in xslt 1.

Reply:
ok thx for the clarification

Reply:
@fluetze applied requested changes

Reply:
How about that?
- limits search for date to type host
- will also find a date in the host of host etc.
But... there are cases where even that will fail

Reply:
The recursive version now prevents that a date is taken from any related item that contains a related item type host, 
e.g.
```
<mods:relatedItem type="foo">
  <mods:relatedItem type="host">
  ...
```  
Those will not match any more...